docs/pages/admin-guides/management/security/proxy-protocol.mdx

```diff
@@ -97,13 +101,21 @@ the Teleport configuration file:
 ```yaml
 proxy_service:
   proxy_protocol: on | off
+  proxy_protocol_allow_downgrade: on | off
   # ...
 auth_service:
   proxy_protocol: on | off
 ```
 
 By default,  `proxy_protocol` is unspecified. Users can manually set
 `proxy_protocol` to `on` or `off`:
+- "unspecified (default)": The associated Teleport service does not require the
```

Comment:
If the config file doesn't accept the string value "unspecified", I would remove the quotes here. If the value is actually the string `"unspecified"`, I would use code style to make that clearer